CMakeLists.txt

```diff
@@ -553,7 +554,7 @@ endif()
 
 # Configure OpenVDB Library and ABI versions
 
-if(NOT OPENVDB_BUILD_CORE)
+if(NOT OPENVDB_BUILD_CORE AND (NOT USE_NANOVDB OR NANOVDB_USE_OPENVDB))
```

Comment:
Okay, if we want to fix this properly, how about:

```
set(NEEDS_OPENVDB OFF)

if(OPENVDB_BUILD_AX OR 
   OPENVDB_BUILD_BINARIES OR
   OPENVDB_BUILD_UNITTESTS OR
   OPENVDB_BUILD_HOUDINI_PLUGIN OR
   OPENVDB_BUILD_MAYA_PLUGIN OR
   OPENVDB_BUILD_PYTHON_MODULE OR
   OPENVDB_BUILD_HOUDINI_ABITESTS)
  set(NEEDS_OPENVDB ON)
endif()

if(NOT OPENVDB_BUILD_CORE AND NEEDS_OPENVDB)
```

Just build nanovdb, without vdb support:
`-DOPENVDB_BUILD_CORE=OFF -DOPENVDB_BUILD_BINARIES=OFF -DOPENVDB_BUILD_NANOVDB=ON`

NanoVDB with vdb support, dont build vdb:
`-DOPENVDB_BUILD_CORE=OFF -DOPENVDB_BUILD_NANOVDB=ON -DNANOVDB_USE_OPENVDB=ON`

NanoVDB without vdb support, but build vdb
`-DOPENVDB_BUILD_NANOVDB=ON -DNANOVDB_USE_OPENVDB=ON`

Reply:
This should then allow you to keep the nanovdb options in the nanocmake